SAKPaaS/src/app/components/user-card/user-card.component.scss

```diff
@@ -3,23 +3,101 @@
     display: none;
   }
   position: absolute;
-  top: 45px;
-  right: 30px;
-  border-radius: 10px;
+  top: 0;
+  right: 0;
+  border-radius: 0 0 0 10px;
+  width: 11%;
+  height: 14%;
```

Comment:
There should be no height attribute here.

![image](https://user-images.githubusercontent.com/15220797/80599297-03c9b900-8a2b-11ea-8a52-b3af259f178f.png)

Reply:
Wenn ich die Größe nicht irgendwo festlege kann ich die runden Ecken zwecks Design nicht dranpacken

Reply:
I think it's doable.